src/test/algo/uint_tuple.t.cc

```diff
@@ -23,9 +23,9 @@
 namespace algo {
 namespace {
 
-template <typename T, size_t ... N>
-constexpr void storage_type_test() {
-  static_assert(std::is_same_v<T, typename uint_tuple<N...>::storage_type>);
+template <typename T, size_t... N>
+void storage_type_test() {
+  STATIC_REQUIRE(std::is_same_v<T, typename uint_tuple<N...>::storage_type>);
```

Comment:
What's STATIC_REQUIRE? Is it a catch2 thing? Because I couldn't google it.

Reply:
https://github.com/catchorg/Catch2/blob/master/include/catch.hpp#L280
As you can see, it's `static_assert` with registration on success.